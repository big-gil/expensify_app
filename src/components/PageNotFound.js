import React from "react";
import { Link } from "react-router-dom";

const PageNotFound = () => (
  <div>
    <p>
      404! Page not found. <Link to="/">Go To Home Page</Link>
    </p>
  </div>
);

export default PageNotFound;
