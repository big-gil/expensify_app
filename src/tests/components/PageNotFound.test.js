import React from "react";
import PageNotFound from "../../components/PageNotFound";
import { shallow } from "enzyme";

test("Page Not Found Test 1 - Page Not Found", () => {
  const wrapper = shallow(<PageNotFound />);
  expect(wrapper).toMatchSnapshot();
});
