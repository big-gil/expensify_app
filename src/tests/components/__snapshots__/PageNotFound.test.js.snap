// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Not Found Test 1 - Page Not Found 1`] = `
<div>
  <p>
    404! Page not found. 
    <Link
      replace={false}
      to="/"
    >
      Go To Home Page
    </Link>
  </p>
</div>
`;
